celerdata/assets/dashboards/CelerData.json

```diff
@@ -0,0 +1 @@
a":"per_second(query2)","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:celerdata.fe.txn.reject.count{$host, $endpoint}.as_rate()"},{"name":"query2","data_source":"metrics","query":"sum:celerdata.fe.txn.failed.count{$host, $endpoint}.as_rate()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]}},{"id":150641777262426,"definition":{"title":"Txn Requset Rate on BE/CN","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"being","formula":"per_second(query1)","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}},{"alias":"exec","formula":"per_second(query2)","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}},{"alias":"commit","formula":"per_second(query3)","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}},{"alias":"rollback","formula":"per_second(query4)","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}}],"queries":[{"name":"query1","data_source":"metrics","query":"sum:celerdata.be.txn.request.count{$host, $endpoint, type:begin}.as_rate()"},{"name":"query2","data_source":"metrics","query":"sum:celerdata.be.txn.request.count{$host, $endpoint, type:exec}.as_rate()"},{"name":"query3","data_source":"metrics","query":"sum:celerdata.be.txn.request.count{$host, $endpoint, type:commit}.as_rate()"},{"name":"query4","data_source":"metrics","query":"sum:celerdata.be.txn.request.count{$host, $endpoint, type:rollback}.as_rate()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]}},{"id":8605514229012284,"definition":{"title":"Publish Tasks on BE/CN","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"vertical","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"total","formula":"query1","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}},{"alias":"failed","formula":"query2","number_format":{"unit":{"type":"custom_unit_label","label":"ops/s"}}}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:celerdata.be.engine_requests.count{$host, $endpoint, status:total, type:publish}.as_count()"},{"name":"query2","data_source":"metrics","query":"avg:celerdata.be.engine_requests.count{$host, $endpoint, status:failed, type:publish}.as_count()"}],"response_format":"timeseries","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}]}}]}}],"template_variables":[{"name":"host","prefix":"host","available_values":[],"default":"*"},{"name":"endpoint","prefix":"endpoint","available_values":[],"default":"*"}],"layout_type":"ordered","notify_list":[],"reflow_type":"auto"}
```

Comment:
The dashboard requires a definition and logo. Please follow our best practices guide [here](https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices) and update your dashboard with the required information. Thanks!

Reply:
done